data_reader.py

```diff
@@ -0,0 +1,121 @@
+import csv
+import glob
+import collections
+import os
+
+from weather import Weather
+
+
+class DataReader:
+
+	annual_report = {}
+
+	def print_annual_report(self, report_data):
```

Comment:
Class is named DataReader. But it does parsing and printing too. Change this to appropriate name.